packages/react-theme-sass/sass/borderRadiusTokens.scss

```diff
@@ -0,0 +1,6 @@
+$borderRadiusNone: var(--borderRadiusNone);
```

Comment:
how did you generate these files ? I'm missing the script automation for this in the PR